### PR TITLE
UI Test ExpanderView

### PR DESCRIPTION
**Pull request by GuidoNeele**
Added a UI Test for the ExpanderView. Tested on an Android device.

### CODE REVIEW

**Comment by GuidoNeele** [comment]
Not sure why all commits are still shown but the changed files are now correct. Took me a while to get the UI tests working because I got a new laptop. But in the end it worked.

**Comment by AndreiMisiukevich** [comment]
> Not sure why all commits are still shown but the changed files are now correct. Took me a while to get the UI tests working because I got a new laptop. But in the end it worked.

the main reason is retargeting to another branch.

I rebased on 4.6.0. Can you rebase your branch to 4.6.0 as well?
I think you should update your origin branch firstly. Or just save your changes, remove fork repo and fork my repo again, then make a PR to expander branch.

File: Xamarin.Forms.Core.UITests.Shared/Queries.cs [hunk 1]
```diff
@@ -1,120 +1,115 @@
 ﻿using System;
-using System.CodeDom;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
```
Comment (AndreiMisiukevich) [inline Xamarin.Forms.Core.UITests.Shared/Queries.cs:7]: I think we should not change this file

**Comment by AndreiMisiukevich** [comment]
Probably I will be able to squash and merge. So just discard changes it not related file

File: Xamarin.Forms.Core.UITests.Shared/Queries.cs (hunk 1 above)
Comment (AndreiMisiukevich) [reply Xamarin.Forms.Core.UITests.Shared/Queries.cs:7]: I think this file should be discarded

**Comment by GuidoNeele** [comment]
Added a new pull request

**Pull request closed without merging**